dsm/resource_aws_sobject.go

```diff
@@ -41,12 +41,13 @@ func resourceAWSSobject() *schema.Resource {
 		"**Note**: Once scheduled deletion is enabled, AWS security object can't be modified.\n\n" +
 		"**Deletion of a dsm_aws_sobject**: Unlike dsm_sobject, deletion of a dsm_aws_sobject is not normal.\n\n" +
 		"**Steps to delete a dsm_azure_sobject:**\n" +
-		"   * Enable schedule_deletion as shown in the examples of guides/dsm_azure_sobject.\n" +
-		"   * Enable delete_key_material as shown in the examples of guides/dsm_azure_sobject.\n" +
+		"   * Enable `delete_key_material` as shown in the examples of `Guides/dsm_aws_sobject`.\n" +
+		"   * Enable `schedule_deletion` as shown in the examples of `Guides/dsm_aws_sobject`.\n" +
```

Comment:
Nit: the file path of `Guides/*` should be `guide/*`

Reply:
It actually refers to Fortanix terraform provider in the Terraform registry. [https://registry.terraform.io/providers/fortanix/dsm/latest/docs]

When the user refers to this doc, he will be in the above url. Hence, it should be Guides/dsm_aws_sobject.